mitodl/mitxonline
# Flexible pricing display personalized price

#### Pre-Flight checklist

#### What are the relevant tickets?
https://github.com/mitodl/mitxonline/issues/670

#### What's this PR do?
Displays the course price, with the deducted flex price discount, for a logged in user who has been approved for flexible pricing.

#### How should this be manually tested?

1. Ensure you have the following already created: standard user, course, course run, flexible pricing tier, flexible price that is approved for the course and user.
3. Login to mitxonline
4. From the dashboard, click 'browse our courses'
5. Click on the course card
6. Click 'Enroll now'
7. A modal will display.  You should see the flexible pricing adjusted certificate price in the modal.
8. Click 'No thanks, I'll take the course....'
9. From the dashboard, click 'Course details'
10. You should see the flexible pricing adjusted certificate price in the card.


#### Screenshots (if appropriate)
![Screen Shot 2022-07-20 at 1 25 03 PM](https://user-images.githubusercontent.com/8311573/180044650-d7c0f71a-6f52-49d1-9c23-8168def47131.png)
![Screen Shot 2022-07-20 at 1 24 26 PM](https://user-images.githubusercontent.com/8311573/180044689-1d011d3a-6091-4353-8cc4-770fcf634b3a.png)
